blocks/site-details/renderSiteOverview.js

```diff
@@ -19,11 +19,23 @@ export default async function renderSiteOverview({
 
   container.innerHTML = renderSkeleton('site-overview');
 
+  const aemSidekickIds = ['ccfggkjabjahcjoljmgmklhpaccedipo', 'olciodlegapfmemadcjicljalmfmlehb', 'ahahnfffoakmmahloojpkmlkjjffnial'];
```

Comment:
@Fankhauser-Dominik there's a ready event you can check https://www.aem.live/developer/sidekick-development. Could you see if this option is better since it's the recommended one ?